src/packages/cli/src/args.ts

```diff
@@ -1,130 +1,168 @@
+import { TruffleColors } from "@ganache/colors";
 import yargs from "yargs";
+import { DefaultFlavor, DefaultOptionsByName } from "@ganache/flavors";
 import {
-  DefaultFlavor,
-  DefaultOptionsByName,
-  EthereumFlavorName
-} from "@ganache/flavors";
-import { Definitions } from "@ganache/options";
-
-const COLORS = {
-  Bold: "\x1b[1m",
-  Reset: "\x1b[0m",
-  FgRed: "\x1b[31m",
-  FgYellow: "\x1b[33m"
-};
+  Base,
+  Definitions,
+  YargsPrimitiveCliTypeStrings
+} from "@ganache/options";
+import { Command, CliOptions, Yargs } from "./types";
+import chalk from "chalk";
+import { EOL } from "os";
+import marked from "marked";
+import TerminalRenderer from "marked-terminal";
+
+marked.setOptions({
+  renderer: new TerminalRenderer({
+    codespan: chalk.hex(TruffleColors.porsche),
+    // Disable `unescape` since doesn't work for everything (we just do it ourselves)
+    unescape: false
+  })
+});
+
+const wrapWidth = Math.min(120, yargs.terminalWidth());
+const NEED_HELP = "Need more help? Reach out to the Truffle community at";
+const COMMUNITY_LINK = "https://gitter.im/ConsenSys/truffle";
+
+function unescapeEntities(html: string) {
+  return html
+    .replace(/&amp;/g, "&")
+    .replace(/&lt;/g, "<")
+    .replace(/&gt;/g, ">")
+    .replace(/&quot;/g, '"')
+    .replace(/&#39;/g, "'");
+}
+const highlight = (t: string) => unescapeEntities(marked.parseInline(t));
+const center = (t: string) => " ".repeat((wrapWidth - t.length) >> 1) + t;
 
 export default function (version: string, isDocker: boolean) {
-  let args = yargs.strict();
+  let args = yargs
+    .strict()
+    .usage(chalk`{hex("${TruffleColors.porsche}").bold ${center(version)}}`)
+    .epilogue(
+      chalk`{hex("${TruffleColors.porsche}").bold ${center(NEED_HELP)}}` +
+        EOL +
+        chalk`{hex("${TruffleColors.turquoise}") ${center(COMMUNITY_LINK)}}`
+    );
 
-  for (const [flavor, flavorDefaults] of Object.entries(DefaultOptionsByName)) {
-    const commandAliases = flavor === DefaultFlavor ? ["$0", flavor] : flavor;
+  let flavor: keyof typeof DefaultOptionsByName;
+  for (flavor in DefaultOptionsByName) {
+    const flavorDefaults = DefaultOptionsByName[flavor];
+    let command: Command;
     let defaultPort: number;
     switch (flavor) {
-      case EthereumFlavorName:
-      default: {
+      // since "ethereum" is the DefaultFlavor we don't need a `case` for it
```

Comment:
sure, i dont think `DefaultFlavor` will ever not be `EthereumFlavorName`, but when you'll have:

``` typescript
switch (flavor) {
  case FilecoinFlavorName:
    ...
  case TezosFlavorName:
    ...
  case DefaultFlavor:
    ...
}
```

It just is easier to grok if it was `EthereumFlavorName`. but i don't really care either way

EDIT: but now I see why `DefaultFlavor` makes sense, since you're assigning the default command to it. 🤷